### PR TITLE
Android app config and fixes

### PR DESCRIPTION
**Pull request by KoalaSat**
## What does this PR do?

This is a post Android release PR, fixing some minor issues found so far:
- Load V8 small size: According to https://github.com/Kudo/v8-android-buildscripts/blob/main/README.md#v8-comes-in-4-flavours , we use now the smaller one: https://github.com/Reckless-Satoshi/robosats/pull/267/files#diff-f4c98a72e8d6c80395084321af0804870bb8fbbe16ac3efbac5d86efeac0ee74R20
- Move CSS with webpack: https://github.com/Reckless-Satoshi/robosats/pull/267/files#diff-a7def86cc36a2178b4ee07b000c53d49e04068db4bc85967c0758578a2f37700R51
- Disable wobble when scrolling: https://github.com/Reckless-Satoshi/robosats/pull/267/files#diff-c92a46984ef937c6501f1fa10feebdf928270c5f2fbd7fe7b24aee34812a9c2cR51
- Missing i18n imports
- Request overload: It's not for sure the definitive solution, but this should help a lot to mitigate this issue. The asset image promise is stored and returned again in case the request is pending, limiting the request per asset to just one https://github.com/Reckless-Satoshi/robosats/pull/267/files#diff-35aa11cfb6d388c4d5c62abbc4b3bc0281ea8d19bc1ec800f0a0f47a59fedbfbR59

## Checklist before merging
- [x] If its a frontend feature, I have ran prettier (`cd frontend; npm run format`).

### CODE REVIEW

File: mobile/setup.md [hunk 1]
```diff
@@ -55,6 +56,11 @@ Make sure you have NDK installed : `Tools > SDK Manager > SDK Tools > NDK (Side
 
 ## Launch app builder and
 
+Make sure you have the `build.js` file created on `robosats/mobile/html/Web.bundle/js`, if not, go to `robosats/frontend/`
```
Comment (Reckless-Satoshi) [inline mobile/setup.md:59]: Maybe meant `main.js` here?

**Review by Reckless-Satoshi — APPROVED** [review]
Works as intended. Wobble is gone, languages load as intended and `npm install` runs without forcing. Have not tested much the image load.

Added the full `Web.bundle` folder on the release workflow as an artifact to keep from the frontend build job (repo does not have the CSS anymore). Remains to be seen whether there have been significant weight reduction. Ready to merge and pre-release an APK.

File: mobile/setup.md (hunk 1 above)
Comment (KoalaSat) [reply mobile/setup.md]: yes, changed

**Comment by KoalaSat** [comment]
@Reckless-Satoshi It's weird I ran prettier and didn't get all those changes 🤔

**Comment by Reckless-Satoshi** [comment]
> @Reckless-Satoshi It's weird I ran prettier and didn't get all those changes thinking

I ran prettier on `/mobile` instead of `/frontend` . The PR template has to be updated to include "I have run prettied in `/mobile`"

**Comment by Reckless-Satoshi** [comment]
Trimmed down about ~50Mb not bad. Pre-release APK https://github.com/Reckless-Satoshi/robosats/releases/tag/android-0689377

**Pull request merged**